presto-product-tests/src/main/java/io/prestosql/tests/hive/TestHiveTransactionalTable.java

```diff
@@ -181,10 +181,11 @@ public void testFailAcidBeforeHive3()
 
     private static String hiveTableProperties(TransactionalTableType transactionalTableType, BucketingType bucketingType)
     {
-        return Stream.concat(
-                transactionalTableType.getHiveTableProperties().stream(),
-                bucketingType.getHiveTableProperties().stream())
-                .collect(joining(",", "TBLPROPERTIES (", ")"));
+        ImmutableList.Builder<String> tableProperties = ImmutableList.builder();
+        tableProperties.addAll(transactionalTableType.getHiveTableProperties());
+        tableProperties.addAll(bucketingType.getHiveTableProperties());
+        tableProperties.add("'no_auto_compaction'='true'");
```

Comment:
hive docs mention this in uppercase

Reply:
https://github.com/apache/hive/commit/e46e9f6c83e9f12b63d9572a7cf0c66d97532571

Since https://issues.apache.org/jira/browse/HIVE-8308 they are case-insensitive.

Reply:
I will change it anyway.

Reply:
Both HDP2.6 and HDP3.1 have much higher Hive versions that one started to have HIVE-8303 (released in 1.1.0)